src/AWS.Deploy.Orchestration/RecipeHandler.cs

```diff
@@ -169,6 +172,21 @@ public async Task<HashSet<string>> LocateCustomRecipePaths(string targetApplicat
             return customRecipePaths;
         }
 
+        /// <summary>
+        /// Runs the recipe level validators and returns a list of failed validations
+        /// </summary>
+        public List<ValidationResult> RunRecipeValidators(Recommendation recommendation, IDeployToolValidationContext deployToolValidationContext)
```

Comment:
As long as we're consolidating this (...somewhere...see earlier comment) - could you also update server mode:

https://github.com/aws/aws-dotnet-deploy/blob/8125cac0c78ac7ce33dd0294cdae010ac5d43f01/src/AWS.Deploy.CLI/ServerMode/Controllers/DeploymentController.cs#L553-L558